examples/ios/swift/Migration/Migration.xcconfig

```diff
@@ -0,0 +1,26 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+//OTHER_SWIFT_FLAGS = $(inherited) -DCREATE_EXAMPLES;
+
+//OTHER_SWIFT_FLAGS = $(inherited) -DSCHEMA_VERSION_0;
+//OTHER_SWIFT_FLAGS = $(inherited) -DSCHEMA_VERSION_1;
+//OTHER_SWIFT_FLAGS = $(inherited) -DSCHEMA_VERSION_2;
+//OTHER_SWIFT_FLAGS = $(inherited) -DSCHEMA_VERSION_3;
+//OTHER_SWIFT_FLAGS = $(inherited) -DSCHEMA_VERSION_4;
+OTHER_SWIFT_FLAGS = $(inherited) -DSCHEMA_VERSION_5;
```

Comment:
Why is this starting on the highest version?

Reply:
`-DSCHEMA_VERSION_5` means this is the version we are migrating to using all versions lower than that.

If `-DCREATE_EXAMPLES` is set this is the version that will be created.

What do you think about `TARGET_SCHEMA_VERSION`? Would that clarify what we are doing here?